src/managers/epicenter-channel-manager.js

```diff
@@ -154,7 +154,17 @@ var EpicenterChannelManager = classFrom(ChannelManager, {
         var project = getFromSessionOrError('', 'project', session);
 
         var baseTopic = ['/group', account, project, groupName].join('/');
-        return __super.getChannel.call(this, { base: baseTopic });
+        var channel = __super.getChannel.call(this, { base: baseTopic });
+        var oldsubs = channel.subscribe;
+        channel.subscribe = function (topic, callback, context, options) {
+            var callbackWithoutPresenceData = function (payload) {
+                if (payload.data && payload.data.type !== 'user') {
```

Comment:
i'd move this check (and the similar  one in getPresenceChannel) to a `isPresenceData()` function